plugins/ja3_fingerprint/ja3_fingerprint.cc

```diff
@@ -126,52 +94,43 @@ getIP(sockaddr const *s_sockaddr, char res[INET6_ADDRSTRLEN])
 }
 
 static std::string
-custom_get_ja3(SSL *s)
+custom_get_ja3(SSL *ssl)
```

Comment:
Just checking: can this not be const? I'm assuming the APIs probably take them non-const which prevents that, but it seems a shame since the plugin should function read-only on this stuff.

Reply:
The APIs take them non-const. This is indeed a shame.